include/seastar/http/routes.hh

```diff
@@ -107,7 +110,9 @@ public:
      * First in higher priority
      * @param rule a rule to add
      * @param type the operation type
-     * @return it self
+     * @return itself
```

Comment:
if we want to s/it self/itself/, probably should be consistent. and i agree that "itself" is better.

Reply:
done